spec/synchronizers/bookingsync_portal/write/source_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe BookingsyncPortal::Write::Source do
```

Comment:
There isn't any spec using vcr so I didn't want to brake convention. VCR isn't even configured.